filebeat/harvester/harvester.go

```diff
@@ -35,9 +36,16 @@ var (
 	ErrClosed       = errors.New("reader closed")
 )
 
+type Outlet interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Outlet should have comment or be unexported